README.md

```diff
@@ -143,8 +117,7 @@ Here is an example of script usage:
 ./scripts/start-baker.sh --base-dir baker \
   --tezos-client baker/tezos-client --tezos-node baker/tezos-node \
   --tezos-baker baker/tezos-baker-005-PsBabyM1 --tezos-endorser baker/tezos-endorser-005-PsBabyM1 \
-  --net-addr 10.147.19.192:8732
-  --peer 10.147.19.49:8732 --base-chain carthagenet
+  --net-addr 10.147.19.192:8732 --peer 10.147.19.49:8732
```

Comment:
Should these be the ...006-PsCARTHA variations?

Reply:
hm, you're right